### ISSUE TITLE
Getting module build failed after npm install @material/drawer @material/list

### ISSUE THREAD

**Issue by mofarril** [material-components/material-components-web-codelabs#157]
[25] ./home.scss 1.45 kB {0} [built] [failed] [1 error]
        + 11 hidden modules

    ERROR in ./home.scss
    Module build failed:
        @include shape-mixins.radius(
       ^
          No argument named $component-height.
       ┌──> node_modules\@material\list\_mixins.scss
    536│ ┌     @include shape-mixins.radius(
    537│ │       $radius,
    538│ │       $rtl-reflexive,
    539│ │       $component-height: $height,
    540│ │       $query: $query
    541│ │     );
       │ └─────^ invocation
       ╵
       ┌──> node_modules\@material\shape\_mixins.scss
    30 │   @mixin radius($radius, $rtl-reflexive: false, $query: feature-targeting-functions.all()) {
       │          ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ declaration
       ╵
      node_modules\@material\list\_mixins.scss 536:5             radius()
      node_modules\@material\list\_mixins.scss 536:5             single-line-shape-radius()
      node_modules\@material\drawer\_mixins.scss 304:3           item-shape-radius()
      node_modules\@material\drawer\_mixins.scss 71:5            core-styles()
      node_modules\@material\drawer\common.scss 24:1             @use
      node_modules\@material\drawer\mdc-drawer.scss 23:1         @forward
      node_modules\@material\drawer\mdc-drawer.import.scss 37:1  @import
      stdin 2:9                                                  root stylesheet
          in C:\Users\mofar\Desktop\Z2M\material-components-web-codelabs\mdc-101\starter\node_modules\@material\list\_mixins.scss (line 536, column 5)
     @ multi (webpack)-dev-server/client?http://localhost:8080 ./home.scss

- [ ] 

**Really new to this and SASS, so I'm not sure what to do to fix.**

**Comment by asyncliz** [comment]
I don't believe the codelab has been updated to v7 yet, can you run the following command instead?

`npm install @material/drawer@^6.0.0 @material/list@^6.0.0`

MDC-101's npm install commands should be up to date, but MDC-102 and beyond may not have version numbers in them yet to prevent this sort of issue. If you see an `@material/*` npm install, add `@^6.0.0` to keep to the right version for the codelab currently.
